CONTRIBUTING.md

```diff
@@ -7,20 +7,18 @@ Please note we have a [code of conduct](./CODE_OF_CONDUCT.md), please follow it
 
 ## Pull Request Process
 
-1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
-   build.
-2. Update the README.md with details of changes to the interface, this includes new environment 
+1. Ensure tests pass before submitting a pull request.
+1. For non-trivial changes, ensure new tests are present before submitting a pull request.
+1. Update the README.md with details of changes to the interface, this includes new environment 
    variables, exposed ports, useful file locations and container parameters.
-3. Increase the version numbers in any examples files and the README.md to the new version that this
-   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
-4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
-   do not have permission to do that, you may request the second reviewer to merge it for you.
+1. You may merge the Pull Request in once you have the sign-off of one other developer, or if you 
+   do not have permission to do that, you may request the reviewer to merge it for you.
```

Comment:
Numbering issue here

Reply:
all 1s in markdown is valid, HTML generators will give the proper numbers. This helps for reordering, removing, or adding new items in the list without needing to keep track of what number in the list it should be